src/vm.c

```diff
@@ -103,14 +93,14 @@ int v7_is_error(struct v7 *v7, val_t v) {
 }
 
 V7_PRIVATE val_t v7_pointer_to_value(void *p) {
-  return ((uint64_t) (uintptr_t) p) & ~V7_TAG_MASK;
+  return ((uint64_t)(uintptr_t) p) & ~V7_TAG_MASK;
 }
 
 V7_PRIVATE void *v7_to_pointer(val_t v) {
   struct {
-    uint64_t s:48;
+    uint64_t s : 48;
   } h;
-  return (void *) (uintptr_t) (h.s = v);
+  return (void *)(uintptr_t)(h.s = v);
```

Comment:
@cpq, we should decide to amend our style guide or our codebase w.r.t the issue of space after case. I don't care but consistency is paramount we all know why.

The google c++ style guide is focused on cpp style casts, but it doesn't show a couple of examples of C casts and there is no space after them. 

I think @vrz here was overzealous and fixed part of the existing code to abide the google c++ style guide, am I right?

I think this PR is ok and can be merged but I don't fancy all this gratuitous restyling. Bad style in the new code is fine, I'm not so picky, but cluttering the git blame history with gratuitous and possibly wrong restyling changes is suboptimal.

I'm going to squash this PR and remove spurious re-styling changes. I'm going to close this PR and push it manually.